csharp/src/SeedLang/Runtime/NativeFunction.cs

```diff
@@ -40,16 +43,77 @@ internal class NativeFunction : IFunction {
 
   // The static class to define all the build-in native functions.
   internal static class NativeFunctions {
+    public const string HasNext = "hasnext";
+    public const string Iter = "iter";
+    public const string Len = "len";
+    public const string List = "list";
+    public const string Next = "next";
+    public const string Range = "range";
```

Comment:
Python:
```
>>> list(range(5))
[0, 1, 2, 3, 4]
```

SeedPython:
```
>>> list(range(5))
---------- Source ----------
list(range(5))

---------- Run ----------
list(range(5))
Eval result: [range(0, 5, 1)]
```

It seems SeedPython doesn't evaluate the range when casting it to a list.

Reply:
Fixed by change the behavior of the list build-in function.